shark/torch_mlir_utils.py

```diff
@@ -56,6 +56,7 @@ def get_torch_mlir_module(
     input: tuple,
     dynamic: bool,
     jit_trace: bool,
+    return_str: bool,
```

Comment:
Let's set this parameter default to `False`. Otherwise, users have to specify it everywhere.

Reply:
Done.